src/GameLoop.tsx

```diff
@@ -554,3 +583,21 @@ export function GameLoop() {
         </div>
     );
 }
+
+export const CountdownOverlay = React.memo(
+    ({ isVisible, countdownComponentIndex }: { isVisible: boolean, countdownComponentIndex: number }) => {
+        const divStyle = {
+            visibility: isVisible ? "visible" as const : "hidden" as const,
+            position: 'absolute',
+            top: '35%',
+            left: '50%',
+            transform: 'translate(-50%, -50%)',
+            zIndex: 2,
+            color: 'red',
+            fontSize: '200%',
+        };
+        return <div style={divStyle}>
+            {countdownComponentIndex}
```

Comment:
What's `countdownComponentIndex`?

Reply:
It is the countdown integer. Index may be a misleading name. Renamed it

Reply:
Still confused.